assets/style.css

```diff
@@ -206,6 +207,10 @@ footer {
   border-radius: 0 0 5px 5px;
 }
 
+.resources:hover {
+  color: #ff0037;
```

Comment:
Try to use the same color as the text around it which is `var(--pink)` instead of `#ff0037`. Or go to a contrasting color. Having two close Shades of pink doesn't look very nice in my opinion.